modules/021-cni-cilium/images/safe-agent-updater/src/main.go

```diff
@@ -52,6 +57,22 @@ func main() {
 	if len(desiredAgentImageHash) == 0 {
 		log.Fatalf("[SafeAgentUpdater] Failed to get env CILIUM_AGENT_DESIRED_IMAGE_HASH.")
 	}
+
+	if !isCiliumExistOnNode() {
+		log.Infof("[SafeAgentUpdater] Cilium CNI binary does not exist on node %s.", nodeName)
```

Comment:
Add both annotations here to node.

Reply:
And below, everywhere, where we put annotation, there should be both annotations.